packages/react-components/src/__tests__/utils.test.tsx

```diff
@@ -51,3 +52,24 @@ describe('isInternalLink', () => {
     });
   });
 });
+
+describe('getIconFromUrl', () => {
+  describe.each`
+    title                | href
+    ${'Google Drive'}    | ${`https://drive.google.com/wrong`}
+    ${'Protocols'}       | ${`https://protocols.io/wrong`}
+    ${'Slack'}           | ${`https://asap.slack.com/wrong`}
+    ${'Google Calendar'} | ${`http://calendar.google.com/r/calendar?12w3`}
+  `('for an $href to render svg with title $title', ({ href, title }) => {
```

Comment:
I don't think this is a sentence :D

Reply:
Also it's strange that this is a describe that synchronously tests things, how does this not cause any lint error?